### PR TITLE
Chore: Display remaining tasks when a task assignments API request fails to fetch

### PR DESCRIPTION
**Pull request by pramodsum**
Fixes merge conflicts from 3e35e3f4

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @pramodsum has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #410

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js [hunk 1]
```diff
@@ -488,7 +500,7 @@ class ActivityFeed extends React.Component<Props, State> {
                         <EmptyState isLoading={isLoading} showCommentMessage={showApprovalCommentForm} />
                     ) : (
                         <ActiveState
-                            inlineError={activityFeedError}
+                            inlineError={activityFeedError ? this.createActivityFeedApiError() : undefined}
```
Comment (DanDeMicco) [inline src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js]: Can you put all this into ```createActivityFeedApiError``` so the ternary isn't needed? Then create a basic UT for it for code coverage :)

File: src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ type Props = {
     versions?: FileVersions,
     comments?: Comments,
     tasks?: Tasks,
-    activityFeedError?: InlineError,
+    activityFeedError?: Errors,
```
Comment (DanDeMicco) [inline src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js]: Will ```activityFeedError``` ever be a ```MaskError```?

File: src/components/ContentSidebar/ActivityFeed/activity-feed/__tests__/__snapshots__/ActiveState-test.js.snap [hunk 3]
```diff
@@ -0,0 +1,76 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (DanDeMicco) [inline src/components/ContentSidebar/ActivityFeed/activity-feed/__tests__/__snapshots__/ActiveState-test.js.snap]: 💯

**Pull request merged**